src/Uno.UI/Controls/CommandBarRenderer.Android.cs

```diff
@@ -194,7 +194,9 @@ protected override void Render()
 				Native.Menu.Clear();
 				foreach (var command in Element.PrimaryCommands.Concat(Element.SecondaryCommands).OfType<AppBarButton>())
 				{
+#pragma warning disable 618
```

Comment:
should this be replaced by something else eventually?

Reply:
ditto for all the rest of the `Obsolete` warning disables. There are quite a few of them...

Reply:
It should, yet until this does not break, it's not a good reason to remove it.